build.sbt

```diff
@@ -61,7 +61,7 @@ val commonLib = (project in file(s"$appsFolder/common-lib"))
     commonSettings,
     libraryDependencies ++= Seq(
       // @todo – we're repeating ourselves. Can we derive this from the plugin?
-      "com.typesafe.play" %% "play" % "2.8.11",
+      "com.typesafe.play" %% "play" % "2.8.19",
```

Comment:
Can you update https://github.com/guardian/typerighter/blob/main/project/plugins.sbt#L1? Play is a strange dependency in that it is normally included as an SBT plugin rather than a library dependency.  Not sure why it is included both ways in this project, but they should be kept in sync.